cluster/selfplay/cc-Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM nvidia/cuda:9.0-cudnn7-devel-ubuntu16.04
 
-RUN apt-get update
-RUN apt-get install -y --no-install-recommends \
+RUN apt-get update && apt-get install -y --no-install-recommends \
         build-essential \
+        bc \
```

Comment:
what's this?

Reply:
basic calculator :)

Reply:
do you still need it?

Reply:
no, but i didn't want to rebuild tensorflow to get rid of it XD.